src/aiida/common/log.py

```
@@ -184,34 +213,32 @@ def configure_logging(with_orm: bool = False, daemon: bool = False, daemon_log_f
             except ValueError:
                 pass
+    # Always add a client log file handler so that all log output is persisted to disk for debugging.
+    try:
+        from aiida.manage.configuration import get_config, get_profile
+
+        profile = get_profile()
+        if profile is not None:
+            log_file = get_config().filepaths(profile)['profile']['log']
+            _add_log_file_handler(config, 'profile_log_file', log_file)
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)